packages/tokens/properties/font/base.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
these are from #207 -- still planning to land separately, but I had to update the base branch since I split the linaria/twin.macro imports to another PR